ansible/ansible_playbook.yml

```diff
@@ -0,0 +1,69 @@
+- name: Run To-Do App
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: trello_api_key
+      prompt: What is the Trello API key?
+    - name: trello_api_token
+      prompt: What is the Trello API token?
+    - name: trello_board_id
+      prompt: What is the Trello board id?
+    - name: trello_todo_list_id
+      prompt: What is the Trello todo list id?
+    - name: trello_doing_list_id
+      prompt: What is the Trello doing list id?
+    - name: trello_done_list_id
+      prompt: What is the Trello done list id?
```

Comment:
Nice work extracting these 👍 I know sometimes it feels overkill for the non-sensitive values, but nice to make them easy to adapt to different environments, and we can always add default values here if they're a pain to enter each time!